### ISSUE TITLE
importance problem

### ISSUE THREAD

**Issue by wsefw** [lijiaqi/HALRP#2]
for i, p in importance.items():
                p += torch.mean(self.param_track[str(i)].grad**2) * len(inputs) / len(dataloader)
The code in the file seems to be based on the gradient of Wbase. According to your article, this should use Wfree instead. Could you please explain why this is the case? I hope my feedback is correct, and I would appreciate it if you could provide feedback after reviewing this message.

**Comment by lijiaqi** [comment]
Thanks for the reminder. This error has been fixed in the latest commit.
